src/components/Shutdowns/ShutdownDetails.tsx

```diff
@@ -59,14 +43,19 @@ const ShutdownDetails: React.FunctionComponent<ShutdownDetailsProps> = ({
       <div className={`flex flex-row justify-between pb-3 ${cardStyles}`}>
         <div className="flex flex-col items-start">
           <div className="text-base md:text-2xl items-center flex flex-row dark:text-white">
-            <h3>{`${shutdown.start_station ? displayStationName(shutdown.start_station) : start_station} - ${shutdown.end_station ? displayStationName(shutdown.end_station) : end_station}`}</h3>
+            <h3>{`${displayStationName(shutdown.start_station)} - ${displayStationName(shutdown.end_station)}`}</h3>
```

Comment:
I was a bit confused why this logic was here; every shutdown has a `start_station` and `stop_station`, but the type had them as optional. Maybe because we were building that type here manually and the station names were coming straight from the query params?

With my refactor to pass in the full `Shutdown` as a prop, we push that concern out of this component, so I think it's fine to update the type to make `start_station` and `end_station` required and remove this `undefined` handling.